lib/model/model.go

```diff
@@ -765,26 +831,10 @@ func (m *model) Completion(device protocol.DeviceID, folder string) FolderComple
 		need.Bytes = 0
 	}
 
-	needRatio := float64(need.Bytes) / float64(tot)
-	completionPct := 100 * (1 - needRatio)
+	comp := NewFolderCompletion(counts.Bytes, need.Bytes, counts.Files+counts.Directories+counts.Symlinks, need.Files+need.Directories+need.Symlinks, need.Deleted)
```

Comment:
Almost feel like this should be NewFolderCompeltion(global, need) and the summing happens internally.

Reply:
yes